pages/attacks/Session_fixation.md

```diff
@@ -25,7 +25,7 @@ ID, and then hijacking the user-validated session by the knowledge of
 the used session ID. The attacker has to provide a legitimate Web
 application session ID and try to make the victim's browser use it.
 
-The session fixation attack is a class of [Session
+The session fixation attack is not a class of [Session
 Hijacking](Session_hijacking_attack "wikilink"), which steals the
```

Comment:
Could you tweak this too since we're here :wink:
```suggestion
Hijacking](Session_hijacking_attack), which steals the
```

Reply:
The link is good, the alt or tooltip text or whatever is a leftover from the automation migration process and can be dumped.

Reply:
Sorry for my ignorance but do I need to submit a new pull request for those changes?

Reply:
No you should be able to accept or commit the change from the comment above.

I think I can do it if you don't see it.

Reply:
The commit suggestion button is greyed out because Github says it's an "Outdated suggestion".

Reply:
You must have done it 😀